### PR TITLE
Adding k8s-build-version as metadata information to finished.json

### PR DESCRIPTION
**Pull request by Rajalakshmi-Girish**
This PR is to add a `metadata.json` file, to artifacts folder in logs with the information about `k8s-build-version` used for periodic ppc64le conformance jobs.

The podutils takecare of merging the` metadata.json` file into `finished.json`.

This new key in `finished.json` enables us to customize column header in test-grid with `k8s-build-version` and its value.
https://github.com/kubernetes/test-infra/blob/master/testgrid/config.md#column-headers

### CODE REVIEW

**Comment by ltccci** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/ppc64le-cloud/test-infra/pull/100#" title="Author self-approved">Rajalakshmi-Girish</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign **mkumatag** after the PR has been reviewed.
You can assign the PR to them by writing `/assign @mkumatag` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=ppc64le-cloud%2Ftest-infra).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/ppc64le-cloud/test-infra/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by Rajalakshmi-Girish** [comment]
//cc @mkumatag

File: config/jobs/periodic/kubernetes/test-kubernetes-periodics.yaml
Comment (mkumatag) [inline config/jobs/periodic/kubernetes/test-kubernetes-periodics.yaml:97]: I see typo in the variable name used! `$K8S-BUILD-VERSION`, it should be `$K8S_BUILD_VERSION`

File: config/jobs/periodic/kubernetes/test-kubernetes-periodics.yaml
Comment (mkumatag) [inline config/jobs/periodic/kubernetes/test-kubernetes-periodics.yaml:178]: same as above

**Review by mkumatag — CHANGES_REQUESTED** [review]

File: config/jobs/periodic/kubernetes/test-kubernetes-periodics.yaml
Comment (mkumatag) [inline config/jobs/periodic/kubernetes/test-kubernetes-periodics.yaml:97]: It is not working!
```
╰─$ K8S_BUILD_VERSION=123 jq --arg key0 'k8s-build-version' --arg value0 $K8S_BUILD_VERSION '. | .[$key0]=$value0' <<<'{}' > metadata.json
╭─manjunath@Manjunaths-MacBook-Pro-2 ~
╰─$
╭─manjunath@Manjunaths-MacBook-Pro-2 ~
╰─$
╭─manjunath@Manjunaths-MacBook-Pro-2 ~
╰─$ cat metadata.json
{}
```

**Review by mkumatag — CHANGES_REQUESTED** [review]

File: config/jobs/periodic/kubernetes/test-kubernetes-periodics.yaml
Comment (Rajalakshmi-Girish) [reply config/jobs/periodic/kubernetes/test-kubernetes-periodics.yaml]: That was in hurry while I replaced underscore with hypen in the key :(

File: config/jobs/periodic/kubernetes/test-kubernetes-periodics.yaml
Comment (Rajalakshmi-Girish) [reply config/jobs/periodic/kubernetes/test-kubernetes-periodics.yaml:97]: I tested it and even checked with running test-pj.sh.. hmm will check again.

Comment (Rajalakshmi-Girish) [reply config/jobs/periodic/kubernetes/test-kubernetes-periodics.yaml:97]: [root@rajalakshmi-workspace test-infra]# export K8S_BUILD_VERSION=123
[root@rajalakshmi-workspace test-infra]# jq --arg key0 'k8s-build-version' --arg value0 $K8S_BUILD_VERSION '. | .[$key0]=$value0' <<<'{}' > metadata.json
[root@rajalakshmi-workspace test-infra]# cat metadata.json
{
  "k8s-build-version": "123"
}
[root@rajalakshmi-workspace test-infra]#

Comment (mkumatag) [reply config/jobs/periodic/kubernetes/test-kubernetes-periodics.yaml:97]: It is working fine,

```shell
╭─manjunath@Manjunaths-MacBook-Pro-2 ~
╰─$ export K8S_BUILD_VERSION=1.2.3
╭─manjunath@Manjunaths-MacBook-Pro-2 ~
╰─$ jq --arg key0 'k8s-build-version' --arg value0 $K8S_BUILD_VERSION '. | .[$key0]=$value0' <<<'{}' > metadata.json
╭─manjunath@Manjunaths-MacBook-Pro-2 ~
╰─$ cat metadata.json
{
  "k8s-build-version": "1.2.3"
}
╭─manjunath@Manjunaths-MacBook-Pro-2 ~
╰─$
```

**Comment by ltccci** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/ppc64le-cloud/test-infra/pull/100#issuecomment-747470777" title="Approved">mkumatag</a>*, *<a href="https://github.com/ppc64le-cloud/test-infra/pull/100#" title="Author self-approved">Rajalakshmi-Girish</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=ppc64le-cloud%2Ftest-infra).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/ppc64le-cloud/test-infra/blob/master/OWNERS)~~ [mkumatag]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by ltccci** [comment]
@Rajalakshmi-Girish: Updated the `job-config` configmap in namespace `prow` at cluster `default` using the following files:
 - key `test-kubernetes-periodics.yaml` using file `config/jobs/periodic/kubernetes/test-kubernetes-periodics.yaml`

<details>

In response to [this](https://github.com/ppc64le-cloud/test-infra/pull/100):

>This PR is to add a `metadata.json` file, to artifacts folder in logs with the information about `k8s-build-version` used for periodic ppc64le conformance jobs.
>
>The podutils takecare of merging the` metadata.json` file into `finished.json`.
>
>This new key in `finished.json` enables us to customize column header in test-grid with `k8s-build-version` and its value.
>https://github.com/kubernetes/test-infra/blob/master/testgrid/config.md#column-headers


Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository.
</details>

**Pull request merged**
